src/control/TrapezoidalVelocityProfile.cpp

```diff
@@ -0,0 +1,35 @@
+#include "TrapezoidalVelocityProfile.h"
+
+using navtypes::Vectord;
+
+namespace control {
+
+SingleDimTrapezoidalVelocityProfile::SingleDimTrapezoidalVelocityProfile(double maxVel,
+																		 double maxAccel)
+	: profile(maxVel, maxAccel) {}
+
+void SingleDimTrapezoidalVelocityProfile::reset() {
+	profile.reset();
+}
+
+void SingleDimTrapezoidalVelocityProfile::setTarget(robot::types::datatime_t currTime,
+													double startPos, double endPos) {
+	Vectord<1> startVec{startPos};
+	Vectord<1> endVec{endPos};
+	profile.setTarget(currTime, startVec, endVec);
+}
+
+bool SingleDimTrapezoidalVelocityProfile::hasTarget() const {
+	return profile.hasTarget();
+}
+
+double
```

Comment:
Why is double in a line by itself? Or is this another git issue from my end?

Reply:
The rest of the function signature is so long, and since there's only 1 parameter it can't split those, so the clang-format decided that this was the best way to limit the line length. I'd say it's pretty clear and since it follows the clang-format rules, it's probably good enough?